### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by serdg0**

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
-    User: <%= user.username %>
-  <% end %>
+    Post Author: <%= user.username %>
+  <% end %><br>
+   Reactions:
+  <%= post.likes %><br><br>
+  Content: <br>  <%= post.content %> <br><br>
+
+  <% post.comments.each do |comment| %>
+  comment author: <%= User.where(id: comment.author_id).pluck(:username).first %><br>
```
Comment (samgaco) [inline app/views/posts/index.html.erb]: - [x] **[REQUIRED]** Try to keep queries in the models (users model in this case), you can style it as a scope or an association for example.

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @MephistoDevelop and @serdg0 :high_brightness: 

Before proceeding we have the following points to address;

- [x] Users should be able to comment on posts for this milestone (right now there is no option for doing so)

> fbcomments


- [x] All the test should be passing (now there are some failed test), if some of those tests are testing   features that are yet to be implemented you can use [pending](https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples), example;

```
describe "an example" do
  it "is implemented but waiting" do
    pending("something else getting finished")
    this_should_not_get_executed
  end
end
```

- [x] All the queries in views should be actually seen in the models, not in the views


------------------------------

moving on :arrow_down: 

* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

-----------------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Comment by MephistoDevelop** [comment]
Fixed:
- [Add Comments in a posts]
 - [ Fixed test to green]
- [Add helpers to do querys and used in views]

testsfakebook

**Review by Oghenebrume50 — APPROVED** [review]
**STATUS:   APPROVED** :+1: 
Hi Team, good job, all issues were worked on and they passed :smile:, now just an optional suggestion.

[OPTIONAL], consider making comment available when a user views an individual post, currently commenting is only possible when viewing all posts. This is implementation details but a very good one

Please move on to the next milestone and code away. 

**RAPHAEL NORIODE**

**Pull request merged**